OpenOversight/app/templates/list_officer.html

```diff
@@ -132,50 +189,81 @@ <h3 class="panel-title accordion-toggle">Age range</h3>
-        <li class="list-group-item">
-            <div class="row">
-              <div class="col-md-6 col-xs-12">
-                <a href="{{ url_for('main.officer_profile', officer_id=officer.id) }}">
-                  <img src="{{ officer.image | default('/static/images/placeholder.png') }}" class="img-responsive thumbnail" alt="{{ officer.full_name() }}">
-                </a>
-              </div>
-              <div class="col-md-6 col-xs-12">
-                <h2>
-                  <a href="{{ url_for('main.officer_profile', officer_id=officer.id) }}" id="officer-profile-{{ loop.index }}">{{ officer.full_name() }}</a>
-                  <small>{{ officer.badge_number()|default('') }}</small>
-                </h2>
-                <div class="row">
-                  <div class="col-md-6 col-xs-6">
-                    <dl>
-                      <dt>Job Title</dt>
-                      <dd>{{ officer.job_title()|default('Unknown') }}</dd>
-                      <dt>Race</dt>
-                      <dd>{{ officer.race_label()|default('Unknown')|lower|title }}</dd>
-                    </dl>
-                  </div>
-                  <div class="col-md-6 col-xs-6">
-                    <dl>
-                      <dt>Gender</dt>
-                      <dd>{{ officer.gender_label()|default('Unknown') }}</dd>
-                      <dt>Number of Photos</dt>
-                      <dd>{{ officer.face.count() }}</dd>
-                    </dl>
+      <div class="search-results-controls top">
+        <div class="top-sort search-results-control">
+          <form class="form-inline">
+            <div class="sort form-group">
+              <label for="sort-results">Sort by:</label>
+              <select id="sort-results" class="form-control">
+                <option value="0"{% if form_data['order'] == 0 %} selected{% endif %}>Last name alphabetical</option>
+                <option value="1"{% if form_data['order'] == 1 %} selected{% endif %}>Rank</option>
+                <option value="2"{% if form_data['order'] == 2 %} selected{% endif %}>Total pay</option>
+                <option value="3"{% if form_data['order'] == 3 %} selected{% endif %}>Salary</option>
+                <option value="4"{% if form_data['order'] == 4 %} selected{% endif %}>Overtime pay</option>
+              </select>
+              <script>
+                $('#sort-results').on('change', function() {
+                  if (window.location.href.includes('order=')) {
+                    window.location.href = window.location.href.replace(/order=[^&]+/, 'order=' + this.value)
+                  } else {
+                    if (window.location.href.includes('?')) {
+                      window.location.href = window.location.href + '&order=' + this.value;
+                    } else {
+                      window.location.href = window.location.href + '?order=' + this.value;
+                    }
+                  }
```

Comment:
This works! If we want to simplify the code we could update it to use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) as pointed out here: https://stackoverflow.com/a/41542008 Seems to be supported by everything except IE now.